src/robot/output/logger.py

```diff
@@ -116,7 +116,7 @@ def _log_message(self, msg):
         """Log messages written (mainly) by libraries"""
         for logger in self._loggers.all_loggers():
             logger.log_message(msg)
-        if msg.level == 'WARN':
+        if msg.level == 'WARN' or msg.level == 'ERROR':
```

Comment:
`msg.level in ('WARN', 'ERROR')` reads a bit better than the above. According to `timeit` it is slightly faster too.